pkg/web/static/openapi.json

```diff
@@ -1275,551 +1292,824 @@
+        "description": "The crypto address object.",
         "properties": {
-          "decrypt": {
-            "type": "boolean",
-            "default": false
+          "id": {
+            "type": "string",
+            "format": "ULID",
+            "description": "The unique identifier of the crypto address."
           },
-          "archives": {
-            "type": "boolean",
-            "default": false
+          "customer_id": {
+            "type": "string",
+            "description": "The customer ID associated with the crypto address."
+          },
+          "payment_system": {
+            "type": "string",
+            "description": "The payment system of the crypto address."
+          },
+          "address_type": {
+            "type": "string",
+            "description": "The type of the crypto address."
+          },
+          "address": {
+            "type": "string",
+            "description": "The crypto address."
+          },
+          "metadata": {
+            "type": "object",
+            "description": "Additional metadata associated with the crypto address."
+          },
+          "created": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The date and time when the crypto address was created."
+          },
+          "modified": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The date and time when the crypto address was last modified."
           }
         }
       },
-      "EnvelopesList": {
+      "CryptoAddressList": {
         "type": "object",
+        "description": "The response object for the crypto addresses endpoint.",
         "properties": {
           "page": {
             "$ref": "#/components/schemas/PageQuery"
           },
-          "is_decrypted": {
-            "type": "boolean"
-          },
-          "secure_envelopes": {
-            "type": "array",
-            "items": {
-              "$ref": "#/components/schemas/SecureEnvelope"
-            }
-          },
-          "decrypted_envelopes": {
+          "crypto_addresses": {
             "type": "array",
             "items": {
-              "$ref": "#/components/schemas/DecryptedEnvelope"
+              "$ref": "#/components/schemas/CryptoAddress"
             }
           }
         }
       },
-      "Counterparty": {
+      "Transaction": {
         "type": "object",
+        "description": "The transaction object.",
         "properties": {
           "id": {
             "type": "string",
-            "format": "ULID",
-            "example": "01HV1GBK4JG9KF8RHJG2KF0B9M"
+            "format": "UUID",
+            "description": "The unique identifier of the transaction."
           },
           "source": {
             "type": "string",
-            "readOnly": true,
-            "enum": [
-              "user",
-              "gds"
-            ]
+            "description": "The source of the transaction."
```

Comment:
```suggestion
            "description": "The source of the transaction. It may be "local" meaning the transaction was created by the user, or "remote" meaning it is an incoming message."
```